st2auth/st2auth/backends/base.py

```diff
@@ -28,10 +30,31 @@ class BaseAuthenticationBackend(object):
     Base authentication class.
     """
 
+    # Capabilities offered by the auth backend
+    CAPABILITIES = (
+        AuthBackendCapability.CAN_AUTHENTICATE_USER
```

Comment:
This is done for backward compatibility reasons (so we don't need to update each existing backend).

Having said that, I also plan to update each existing backend in the near future to explicitly spell out capabilities offered by it.